app/locales/en.json

```diff
@@ -6,11 +6,19 @@
     "authorities_desc": "Choose trusted healthcare authorities in your area to obtain exposure data. Either select a name from the global registry, or enter the web address provided by an authority which has implemented Safe Paths.",
     "authorities_input_placeholder": "Paste your URL here",
     "authorities_no_sources": "No data source yet",
+    "authorities_new_in_area_title": "New Healthcare Authority In Your Region",
+    "authorities_new_in_area_msg": "Subscribe to {{numAuthorities}} new trusted Healthcare Authority in your location",
+    "authorities_new_in_area_msg_plural": "Subscribe to {{numAuthorities}} new trusted Healthcare Authorities in your location",
+    "authorities_new_subcription_title": "{{numAuthories}} New Subscription",
```

Comment:
move these out of `label.x` and into a logical group. `<whatever you think>.x` 

label has become useless and a dumping ground

lokalise handles this nicely.

run `yarn i18n:extract` to diff en.json changes. make sure there are no extraneous keys here. but don't commit any empty strings.

Reply:
If it's alright with you, I'd like to just leave these as is and come back with another PR to re-organize this entire file. I think we should try to settle on a convention on what to break strings down by, e.g. file, functionality, etc.

Reply:
Normally I'd say "yep sure" but I'm 50% sure this will create double work for 12 translators when we rename keys later.

Granted... There's a lot of key renaming warranted.... So we need a solution that does not interrupt translators... 

Ok.. seems fine to leave like this!